.github/workflows/autocounterd.yml

```diff
@@ -4,10 +4,6 @@ on:
   push:
     branches:
       - "master"
-    paths:
```

Comment:
Why remove?

Reply:
autocounterd binary depends on gnoclient, vm, rpc/client, commands, so if those change this needs to change too

Reply:
this is just for the docker build, so i'm actually ok in reverting this seeing as it's otherwise tested in misc.yml